### ISSUE TITLE
[Bug]: copy pet

### ISSUE THREAD

**Issue by Utopia1916** [Fuzss/moblassos#5]
### Mod Loader (Required)

Fabric

### Minecraft Version(s) (Required)

1.19.4

### Mod Version(s) (Required)

6.0.0

### Minimal Setup (Required)

Yes

### Notes (Required)

I conducted the act by wielding a golden cord, applying it to the tamed pet, which remained in its original spot. The cord in hand commenced a countdown, only to be employed once more, and then，omg... heavens above, this is utterly extraordinary; my pet has been replicated XD, with the potential for multiple reproductions, perhaps even an infinite replication?

### latest.log (Required)

https://gist.github.com/Oururis/cb1a1f3b6e5fc236ebc4b7f94602f3c2

**Comment by Fuzss** [comment]
A little more information would be great, like what type of pet, what lasso you used. Maybe a video demonstrating the bug if possible.

And do you have any other mods related to pets, like something that would prevent them from dying?

**Comment by Utopia1916** [comment]
Certainly, I can record a video and upload it to YouTube. I will send the link later. I have made no other alterations related to the pet, although the pet teleportation by waystone could be considered a slight modification, perhaps?


YT link:https://youtu.be/0Qs1wzgW9HE
You will be able to view the video shortly; it is currently undergoing inspection.

I have tested wolves and cats, and only the cat was copyed. I can confidently say that there is definitely a conflict with one of the mods I am using, but pinpointing it is proving to be quite challenging due to the extensive list of 199 mods I have. 

I should mention that I have scarcely employed any mods that alter creatures.

If you want, I can export the current version of my HMCL modpack.


I'm not sure if you can use PIKPAK, but I've uploaded it first （If you want to specify using a different cloud storage service, I'll try to accommodate that as much as possible.）. if u want ,you can open the link, then log in with Google, save it to your own cloud storage. After that, you should be able to download this modpack in Chrome, which includes all my mods, settings, config, and so on, except for saves and resourcepacks. （modpack including the launcher）
https://mypikpak.com/s/VNtFcsv1YiNXoRU9HLc8eXQuo1

**Comment by Oururis** [comment]
YT link:https://youtu.be/0Qs1wzgW9HE
You will be able to view the video shortly; it is currently undergoing inspection.

I have tested wolves and cats, and only the cat was copyed. I can confidently say that there is definitely a conflict with one of the mods I am using, but pinpointing it is proving to be quite challenging due to the extensive list of 199 mods I have. Furthermore, there are very few, if any, pet-related mods, and even fewer related to animals.

**Comment by Oururis** [comment]
I should mention that I have scarcely employed any mods that alter creatures.

**Comment by Oururis** [comment]
If you wish, I can export the current version of my HMCL pack.

**Comment by Utopia1916** [comment]
If you need any other help from me, feel free to ask

**Comment by Fuzss** [comment]
Thanks for the demonstration, that makes it more clear.

Unfortunately I'm unable to reproduce the issue with just Mob Lassos, so it must be a mod conflict. Each lasso has an entity type tag for blacklisting certain mobs, I'd recommend putting cats in there to prevent the duplication. You can find the tag locations in the mod jar.

**Comment by Utopia1916** [comment]
Alright, I'll give it a shot

I also tried using only Mob Lassos, and I didn't encounter any duplication issues. However, I couldn't pinpoint the conflicting mod since the mods I'm using don't alter the creatures.

If I discover the conflicting mod in the future, I'll let you know.

**Comment by Utopia1916** [comment]
i found it,https://modrinth.com/mod/dismount-entity
